build.gradle

```diff
@@ -1,6 +1,29 @@
-group 'multi-int-store'
-version '0.1.0-SNAPSHOT'
+plugins {
+	id 'org.springframework.boot' version '2.1.4.RELEASE'
+	id 'java'
+}
+
+apply plugin: 'io.spring.dependency-management'
+
+group = 'com.multiintstore'
+version = '0.0.1-SNAPSHOT'
+sourceCompatibility = 11
```

Comment:
❓ 1.11? If this compiles it may accept both.

Reply:
I've seen examples using 11. It compiles either way, but I think `11` is what you should normally put.